0.4.45: Compilation issue

Hello. If I update to 0.4.45 version, I am seeing this error in compilation:
```bash
error[E0425]: cannot find value `CURLMOPT_MAX_CONCURRENT_STREAMS` in crate `curl_sys`
   --> /github/home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/curl-0.4.45/src/multi.rs:380:36
    |
380 |         self.setopt_long(curl_sys::CURLMOPT_MAX_CONCURRENT_STREAMS, val as c_long)
    |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `curl_sys`
 ```
 Is it something particular to my project?
 
 Have dependencies changed with latest version?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "curl"
-version = "0.4.45"
+version = "0.4.46"
 authors = ["Alex Crichton <alex@alexcrichton.com>"]
 license = "MIT"
 repository = "https://github.com/alexcrichton/curl-rust"
@@ -18,7 +18,7 @@ appveyor = { repository = "alexcrichton/curl-rust" }
 
 [dependencies]
 libc = "0.2.42"
-curl-sys = { path = "curl-sys", version = "0.4.59", default-features = false }
+curl-sys = { path = "curl-sys", version = "0.4.72", default-features = false }
 socket2 = "0.5.0"
 
 # Unix platforms use OpenSSL for now to provide SSL functionality

